fe/fe-core/src/test/java/com/starrocks/sql/analyzer/PrivilegeCheckerTest.java

```diff
@@ -3049,6 +3050,162 @@ public void testStorageVolumeStatement() throws Exception {
                 "Access denied;");
     }
 
+    @Test
+    public void testPipe() throws Exception {
```

Comment:
need to add more ut, e.g., show grants for pipe obj is not tested

Reply:
and also select from information_schema.grants_to_user, grants_to_role, better to use sql tests to cover it, ut is not easy

Reply:
the `information_schema.grants*` should be same as `show grants`, is it necessary to test it again ?